iohub/ngff/nodes.py

```diff
@@ -1010,6 +1010,43 @@ def get_axis_index(self, axis_name: str) -> int:
         """
         return self.axis_names.index(axis_name.lower())
 
+    def get_transforms(
```

Comment:
Since this method does return get the actual transforms (as stored in the JSON files), but the accumulated effective value, I would name it more explicitly. Also does it make sense to split this into two methods so that the return type is more apparent? Something like:

```py
def get_effective_scale(self, image: str | Literal["*"]) -> TransformationMeta: ...
def get_effective_translation(self, image: str | Literal["*"]) -> TransformationMeta: ...
```

If this makes sense, we should also rewire the `scale` property to be a special case for this method to avoid duplication.

Reply:
That was my original plan, but I thought the translation was dependent on the scale. This is where my lack of familiarity with the spec shows.
I thought the translation was dependent on the scale so it made sense to do them bth at once.